Tests/SwiftFormatTests/PrettyPrint/ValueGenericsTests.swift

```diff
@@ -0,0 +1,44 @@
+final class ValueGenericsTests: PrettyPrintTestCase {
+  func testValueGenericDeclaration() {
+    let input = "struct Foo<let n: Int> { static let bar = n }"
+    let expected = """
+      struct Foo<
+        let n: Int
+      > {
+        static let bar = n
+      }
+
+      """
+    assertPrettyPrintEqual(
+      input: input,
+      expected: expected,
+      linelength: 20,
+      experimentalFeatures: ["valueGenerics"]
```

Comment:
Should we use `ExperimentalFeatures` enum cases here instead of spelling out the feature names?

Reply:
Yeah, I guess that's fine for tests since we don't have to be worried about SPI leakage there. Updated.